src/screens/gm/objectCreationView.cpp

```diff
@@ -42,10 +42,14 @@ GuiObjectCreationView::GuiObjectCreationView(GuiContainer* owner)
     std::sort(template_names.begin(), template_names.end());
     for(string template_name : template_names)
     {
-        (new GuiButton(box, "CREATE_STATION_" + template_name, template_name, [this, template_name]() {
-            setCreateScript("SpaceStation():setRotation(random(0, 360)):setFactionId(" + string(faction_selector->getSelectionIndex()) + "):setTemplate(\"" + template_name + "\")");
-        }))->setTextSize(20)->setPosition(-350, y, ATopRight)->setSize(300, 30);
-        y += 30;
+        auto stationTemplate=ShipTemplate::getTemplate(template_name);
+        if (stationTemplate)
```

Comment:
This check looks odd, why wouldn't a template exist if it is in the list of templates?